jetty-server/src/main/java/org/eclipse/jetty/server/HttpConfiguration.java

```diff
@@ -71,6 +73,7 @@
     private long _minRequestDataRate;
     private long _minResponseDataRate;
     private CookieCompliance _cookieCompliance = CookieCompliance.RFC6265;
+    private CookieCompliance _setCookieCompliance = CookieCompliance.RFC6265;
```

Comment:
Having two different CookieCompliance levels seems odd.

If you have a client that talks RFC2965, (and we are configured to understand that client), then talking back on RFC6265 would be bad.